### PR TITLE
evm: ignore precompile addresses for some target access events

### PR DESCRIPTION
**Pull request by g11tech**
address a pending change of ignoring precompiles for some target access events

### CODE REVIEW

File: packages/evm/src/opcodes/gas.ts [hunk 1]
```diff
@@ -612,13 +623,16 @@ export const dynamicGasHandlers: Map<number, AsyncDynamicGasHandler | SyncDynami
...
         gas += subMemUsage(runState, outOffset, outLength, common)
 
         let charge2929Gas = true
-        if (common.isActivatedEIP(6800)) {
-          const toAddress = new Address(addresstoBytes(toAddr))
+        if (
+          common.isActivatedEIP(6800) &&
+          runState.interpreter._evm.getPrecompile(toAddress) === undefined
+        ) {
           // TODO: add check if toAddress is not a precompile
```
Comment (acolytec3) [inline packages/evm/src/opcodes/gas.ts:636]: Can we remove this comment here?

**Review by acolytec3 — APPROVED** [review]
Looks good.  Left one question on a seeming obsolete comment.

File: packages/evm/src/opcodes/gas.ts (hunk 1 above)
Comment (g11tech) [reply packages/evm/src/opcodes/gas.ts:636]: ahh yes lol

**Review by acolytec3 — APPROVED** [review]
LGTM!  @holgerd77 could we admin merge this?  The only test failure is browser tests which are failing for some unknown vitest related reason at the moment.

**Pull request merged**